src/config/constants.js

```diff
@@ -10,4 +10,9 @@ export default {
     INSTA_URL: process.env.REACT_APP_INSTA_URL || 'https://www.instagram.com/codingcoach_io/',
     TWITTER_URL: process.env.REACT_APP_TWITTER_URL || 'https://twitter.com/codingcoach_io',
   },
+  privacyPolicy: {
```

Comment:
What about calling this key `legal`, then we can have *terms & conditions* (If needed) as well as the privacy policy